include/tilck/kernel/test/tracing.h

```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#pragma once
+
+#include <tilck/mods/tracing.h>
+STATIC bool simple_wildcard_match(const char *str, const char *expr);
+STATIC int set_traced_syscalls_int(const char *str);
+extern bool *traced_syscalls;
+STATIC extern char *traced_syscalls_str;
```

Comment:
"STATIC extern" does not make any sense. It's either one or the other. Please check the code.
Also, add a vertical space to separate the functions from the variables.